src/Blazored.Modal/BlazoredModal.razor

```diff
@@ -5,7 +5,7 @@
 
     <div class="blazored-modal-overlay" @onclick="HandleBackgroundClick"></div>
 
-    <div class="blazored-modal">
+    <div class="@Style">
```

Comment:
I think instead of using a single style, I would rather always have the `blazored-modal` style set here. Then any passed in styles added after so something like this.

```suggestion
    <div class="blazored-modal @Style">
```

This is a fairly common pattern in third party components as the consumer has a default set of styles and they can then choose to override one or multiple. Rather than having to possibly recreate the whole default style class just to change one property.